build.gradle.kts

```diff
@@ -115,7 +115,9 @@ sonar {
                 "dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/configMLDT/" +
                 "configForEutaxonomyFinancialsMLDT.ts," +
                 "dataland-frontend/src/frameworks/esg-questionnaire/ViewConfig.ts," +
-                "dataland-frontend/src/frameworks/esg-questionnaire/UploadConfig.ts",
+                "dataland-frontend/src/frameworks/sfdr/ViewConfig.ts," +
+                "dataland-frontend/src/frameworks/esg-questionnaire/UploadConfig.ts" +
+                "dataland-frontend/src/frameworks/sfdr/UploadConfig.ts",
```

Comment:
Gehört nicht in diese Datei ist mir aber gerade aufgefallen: SFDR ist jetzt das erste Framework auf der provide data Seite und in dem Filter dropdown. Das müsste wieder auf die ursprüngliche Reihenfolge geändert werden.

Reply:
Fixed by introducing explicit framework ordering in the backend